TM1py/Services/CellService.py

```diff
@@ -204,18 +222,27 @@ def get_dimension_names_for_writing(self, cube_name: str, **kwargs) -> List[str]
         return dimensions
 
     def write_value(self, value: Union[str, float], cube_name: str, element_tuple: Iterable,
-                    dimensions: Iterable[str] = None, **kwargs) -> Response:
+                    dimensions: Iterable[str] = None, sandbox_name: str = None,**kwargs) -> Response:
         """ Write value into cube at specified coordinates
 
         :param value: the actual value
         :param cube_name: name of the target cube
         :param element_tuple: target coordinates
         :param dimensions: optional. Dimension names in their natural order. Will speed up the execution!
+        :paramm sandbox_name: str
         :return: response
         """
         if not dimensions:
             dimensions = self.get_dimension_names_for_writing(cube_name=cube_name)
-        url = format_url("/api/v1/Cubes('{}')/tm1.Update", cube_name)
+        url = ""
+        if sandbox_name == None:
+            url = format_url("/api/v1/Cubes('{}')/tm1.Update", cube_name)
```

Comment:
@andreyea,
perhaps it's easier if we add an optional `sandbox_name` argument to the `format_url` function.
Then we can outsource the check in the function. What do you think?

Reply:
I guess, it's an option, but it feels a little ambiguous to append extra parameter in there.